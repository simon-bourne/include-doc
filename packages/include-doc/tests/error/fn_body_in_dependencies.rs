use include_doc::function_body;

#[doc = function_body!("code.rs", function, [function])]
fn main() {}
