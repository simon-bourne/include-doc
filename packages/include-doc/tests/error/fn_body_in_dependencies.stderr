error: Function body can't be in dependencies
 --> tests/error/fn_body_in_dependencies.rs:3:35
  |
3 | #[doc = function_body!("code.rs", function, [function])]
  |                                   ^^^^^^^^
